model/trace.go

```diff
@@ -32,3 +32,21 @@ func (t *Trace) NormalizeTimestamps() {
 		span.NormalizeTimestamps()
 	}
 }
+
+func (m *Trace) DenormalizeProcess(p *Process) {
+	for _, v := range m.Spans {
+		v.Process = p
+	}
+}
+
+func (m *Trace) FlattenToSpansMaps(spanMap map[TraceID][]*Span) {
```

Comment:
this is used in a single place, just define the function there, not at the model level

Reply:
same for DenormalizeProcess - it's an absolutely trivial loop, no reason to have a model-level function for it